packages/dds/matrix/test/matrix.undo.spec.ts

```diff
@@ -19,383 +20,325 @@ import { UndoRedoStackManager } from "./undoRedoStackManager";
 
 describe("Matrix", () => {
     describe("undo/redo", () => {
-        describe("local client", () => {
```

Comment:
Refactored the single client undo/redo tests so they can also be run a second time w/multiple attached clients for some cheap additional coverage.